### PR TITLE
publishing

### PR DESCRIPTION
**Pull request by helenaaaaaaaaaa**
I have made some further changes and clarifications.

### CODE REVIEW

File: tutorials/ai-core-metrics/ai-core-metrics.md [hunk 1]
```diff
@@ -483,12 +502,16 @@ For metrics tagged with the artifact name, you can also locate in the **Models**
 
 [OPTION BEGIN [Postman]]
 
+Click through `AI Core` > `lm` > `metrics` > `Get metrics` and double check the `executionId`.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-metrics/ai-core-metrics.md:505]: I think that saying "Navigate  to" sounds better than "Click through".

File: tutorials/ai-core-metrics/ai-core-metrics.md [hunk 2]
```diff
@@ -180,16 +180,21 @@ aic_connection.metrics.log_metrics(
 )
 ```
 
+The parameter `value="housepricemodel"` refers to the artifact name, which references the model that will be stored in AWS S3. It is vital that the name of this parameter matches the name that you defined in your yaml workflow.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-metrics/ai-core-metrics.md:183]: YAML workflow

File: tutorials/ai-core-metrics/ai-core-metrics.md [hunk 3]
```diff
@@ -457,24 +473,27 @@ Create configuration using the following information. The information is taken f
...
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 9: ](Retrieve metrics)]
+[ACCORDION-BEGIN [Step 10: ](Retrieve metrics)]
 
 
 [OPTION BEGIN [SAP AI Launchpad]]
 
-Click on the `Metrics Resource` tab of your execution.
+Click through `ML Operations` >  `Executions` > `Metrics Resource` tab of your execution.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-metrics/ai-core-metrics.md:492]: I think that saying "Navigate to" sounds better than "Click through".

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by helenaaaaaaaaaa** [comment]
I have made the requested changes

**Pull request merged**